expression/builtin_math.go

```diff
@@ -190,8 +192,67 @@ func builtinRound(args []types.Datum, ctx context.Context) (d types.Datum, err e
 
 // See http://dev.mysql.com/doc/refman/5.7/en/mathematical-functions.html#function_conv
 func builtinConv(args []types.Datum, ctx context.Context) (d types.Datum, err error) {
-	//TODO  implement
-	return d, errors.New("Function unimplement")
+	var (
+		n        string
+		r        string
+		toBase   int64
+		fromBase int64
+		signed   bool
+	)
+	for _, arg := range args {
+		if arg.IsNull() {
+			return d, nil
+		}
+	}
+
+	n, err = args[0].ToString()
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+	sc := ctx.GetSessionVars().StmtCtx
+	fromBase, err = args[1].ToInt64(sc)
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+	toBase, err = args[2].ToInt64(sc)
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+
+	if fromBase < 0 {
+		fromBase = -fromBase
+	}
+	if toBase < 0 {
+		signed = true
```

Comment:
Please refer to mysql source code: https://github.com/mysql/mysql-server/blob/5.7/sql/item_strfunc.cc#L3959